utils/wasm-builder/src/wasm_project.rs

```diff
@@ -99,11 +99,16 @@ impl WasmProject {
         lib.insert("name".into(), crate_info.snake_case_name.into());
         lib.insert("crate-type".into(), vec!["cdylib".to_string()].into());
 
+        let mut dev_profile = Table::new();
+        dev_profile.insert("panic".into(), "abort".into());
+
         let mut release_profile = Table::new();
         release_profile.insert("lto".into(), true.into());
         release_profile.insert("opt-level".into(), "s".into());
+        release_profile.insert("panic".into(), "abort".into());
```

Comment:
It's not the best choice, because in future we will be able set "unwind" for wasm, once it's been released.

Reply:
Copied it from the substrate-wasm-builder, but removed it back 😄